src/renderer/base/renderer.cpp

```diff
@@ -1036,7 +1037,10 @@ void Renderer::_PaintOverlay(IRenderEngine& engine,
         // Set it up in a Viewport helper structure and trim it the IME viewport to be within the full console viewport.
         Viewport viewConv = Viewport::FromInclusive(srCaView);
 
-        for (SMALL_RECT srDirty : engine.GetDirtyArea())
+        gsl::span<const til::rectangle> dirtyAreas;
+        LOG_IF_FAILED(engine.GetDirtyArea(dirtyAreas));
+
+        for (SMALL_RECT srDirty : dirtyAreas)

```

Comment:
while you're here, might as well make it `SMALL_RECT&` or even `auto&`

Reply:
It actually can't do the ref way. The right side is `til::rectangle`s so making it the value here has it run the operator to convert it out to a `SMALL_RECT`.

Reply:
erp, got it